web/src/search.js

```diff
@@ -131,16 +152,7 @@ export function initialize({on_narrow_search}) {
             }
 
             if (keydown_util.is_enter_event(e) && $search_query_box.is(":focus")) {
-                // We just pressed Enter and the box had focus, which
-                // means we didn't use the typeahead at all.  In that
```

Comment:
This isn't true anymore -- using enter in the typeahead causes pills to be created. You have to press enter again when the box has focus to trigger a search